indico/modules/events/registration/fields/simple.py

```diff
@@ -329,3 +335,40 @@ def _to_machine_date(date):
 
 def _to_date(date):
     return datetime.strptime(date, '%Y-%m-%d').date()
+
+
+class MultiChoiceField(ChoiceBaseField):
+    name = 'multi_choice'
+    wtf_field_class = wtforms.SelectMultipleField
+
+    @property
+    def wtf_field_kwargs(self):
+        return {'choices': [(id, caption) for id, caption in self.form_item.data['captions'].iteritems()]}
+
+    @classmethod
+    def process_field_data(cls, data, old_data=None, old_versioned_data=None):
```

Comment:
this is the same as for SingleChoiceItem besides the default item logic. i'd rather have a class level `has_default_item` attribute than repeating so much code